server/middleware/one-time-code-auth/setUpOneTimeCode.ts

```diff
@@ -15,10 +15,20 @@ export default function setUpOneTimeCode(app: Express, oneTimeCodeService: OneTi
   const requestOneTimeCodeController = new RequestOneTimeCodeController(oneTimeCodeService, verifyOneTimeCodeController)
   const oneTimeCodeEmailSentController = new OneTimeCodeEmailSentController()
 
+  app.use('/oneTimeCode', (req, res, next) => {
```

Comment:
A little bit of refactoring for the `oneTimeCodeValidityDuration` and `lsjSessionDuration` values.
Previously they were setup in a corresponding view class for the pages that needed them (2 places). We need the `oneTimeCodeValidityDuration` value on the 'code no longer valid' page as well, so to follow that pattern would have meant creating a third view class and a controller for 'code no longer valid' which felt like overkill.
Given that these are both readonly values and are not part of a form submission I decided to remove them from the view classes 👇 and simply add them as locals